lntest/rpc/lnd.go

```
@@ -730,3 +731,17 @@ func (h *HarnessRPC) LookupHtlcResolutionAssertErr(
 
 	return err
 }
+
+// Quiesce makes an RPC call to the node's Quiesce method and returns the
+// response.
+func (h *HarnessRPC) Quiesce(req *devrpc.QuiescenceRequest) (
+	*devrpc.QuiescenceResponse, error) {
+
+	ctx, cancel := context.WithTimeout(h.runCtx, DefaultTimeout)
+	defer cancel()
+
+	res, err := h.Dev.Quiesce(ctx, req)
+	h.NoError(err, "Quiesce")
+
+	return res, err
```

Don't need to return an error if we always assert it's not nil?